CHANGES.rst

```diff
@@ -1,10 +1,11 @@
 Changes
 =======
 
-Version 0.7.1 (UNRELEASED)
+Version 0.7.1 (2020-11-09)
 --------------------------
 
 - Allows ``htcondor_max_runtime` and ``htcondor_accounting_group`` to be specified for HTC jobs.
+- Changes CI system from Travis to GitHub Actions.
```

Comment:
See my comment https://github.com/reanahub/reana-workflow-engine-yadage/pull/170/files#r519923812 regarding HTC and CI phrasing.